<launch>
  <node pkg="localization_ndt"
        type="ndt_localizer_node"
        name="ndt_localizer"
        output="screen">

    <param name="map_pcd_path" value="/home/wr/xw.pcd"/>
    <param name="scan_topic"   value="/Y991B020189BC0018/scan"/>
    <param name="odom_topic"   value="/Y991B020189BC0018/odom/wheel"/>
    <param name="imu_topic"    value="/Y991B020189BC0018/imu"/>

    <!-- 可选：NDT 参数 -->
    <param name="ndt_resolution"        value="1.0"/>
    <param name="ndt_num_threads"       value="4"/>
    <param name="ndt_max_fitness_score" value="2.0"/>
  </node>
</launch>
